app/src/main/java/com/dessalines/thumbkey/keyboards/UARUMessageEaseSymbols.kt

```diff
@@ -781,7 +781,7 @@ val KB_UA_RU_MESSAGEEASE_SYMBOLS_SHIFTED = KeyboardC(
 )
 
 val KB_UA_RU_MESSAGEEASE_SYMBOLS: KeyboardDefinition = KeyboardDefinition(
-    title = "UA RU MessageEase український-русский with Symbols",
+    title = "RU MessageEase русский (RU+UA multi) symbols",
```

Comment:
The ukrainian language code is UK : https://en.m.wikipedia.org/wiki/Ukrainian_language

Reply:
ah, okay. i see - i got confused, because i was using the ISO country codes, and not the language codes. i guess language codes make more sense, due to stuff like esperanto which is not tied to a country. will fix